pom.xml

```
@@ -67,10 +67,10 @@
         <gravitee-alert-api.version>1.9.0</gravitee-alert-api.version>
         <gravitee-cockpit-api.version>1.10.0</gravitee-cockpit-api.version>
         <gravitee-common.version>1.25.0</gravitee-common.version>
-        <gravitee-connector-api.version>1.1.0</gravitee-connector-api.version>
+        <gravitee-connector-api.version>1.1.2</gravitee-connector-api.version>
```

You could use the 1.1.3 version.
Not sure you could use the 1.1.4 though because it declares gravitee-common in 1.27.0
```suggestion
        <gravitee-connector-api.version>1.1.3</gravitee-connector-api.version>
```